lib/let_me/builder.ex

```diff
@@ -129,9 +142,7 @@ defmodule LetMe.Builder do
     combined_condition =
       case {allow_condition, deny_condition} do
         {false, _} -> false
-        {_, true} -> false
```

Comment:
Could you add back the two lines you removed here? I think we may need them. There's a comment in line 127 - I don't see that warning when running Dialyzer in the library repo, but I've seen it under certain conditions when used in applications.